### PR TITLE
[Spec] Local specification of aptos_account, aptos_coin, aptos_governance

### PR DESCRIPTION
**Pull request by UIZorrot**
### Description

This is the local specifcation of aptos_account, aptos_coin, aptos_governance module, including aborts_if and ensures
Some function are commented as 'no need verify', I'm not sure if they really don't need any specifcation.

### Test Plan

aptos move prove -f aptos

### CODE REVIEW

File: aptos-move/framework/aptos-framework/sources/aptos_account.spec.move [hunk 1]
```diff
@@ -217,4 +226,17 @@ spec aptos_framework::aptos_account {
         aborts_if type_info::type_of<CoinType>() != type_info::type_of<AptosCoin>()
             && !coin::is_account_registered<CoinType>(to) && !can_receive_direct_coin_transfers(to);
     }
+
+    spec schema TransferEnsures<CoinType> {
+        coin_store_to: coin::CoinStore<CoinType>;
+        coin_store_source: coin::CoinStore<CoinType>;
+        amount: u64;
+
+        let o_balance_source = coin_store_source.coin.value;
```
Comment (rahxephon89) [inline aptos-move/framework/aptos-framework/sources/aptos_account.spec.move:235]: nit: could you explain the name "o_" and "p_"?

**Review by rahxephon89 — APPROVED** [review]

File: aptos-move/framework/aptos-framework/sources/aptos_account.spec.move (hunk 1 above)
Comment (UIZorrot) [reply aptos-move/framework/aptos-framework/sources/aptos_account.spec.move:235]: Oh, it means the origin and the post, should I add comment?

Comment (rahxephon89) [reply aptos-move/framework/aptos-framework/sources/aptos_account.spec.move:235]: maybe rename to "origin_..." and "post_" to make it more clear?

Comment (UIZorrot) [reply aptos-move/framework/aptos-framework/sources/aptos_account.spec.move]: Fixed

**Pull request merged**